# ELK1

## Overview
ELK1 is a gene that encodes the ETS transcription factor ELK1, a member of the ETS family of transcription factors. The protein plays a critical role in the regulation of gene expression by transducing extracellular signals into transcriptional responses, primarily through the MAP kinase signaling pathway. ELK1 interacts with DNA and other proteins to regulate the transcription of genes involved in various cellular processes, including cell proliferation, differentiation, and response to external stressors. The protein's function is modulated by several post-translational modifications, which affect its stability and activity in the cell. Due to its central role in critical signaling pathways, ELK1 is also implicated in various clinical conditions, including cancer and neurodegenerative diseases, making it a potential target for therapeutic intervention (Boros2009Elucidation; Besnard2011Elk-1).

## Structure
The molecular structure of the ELK1 protein, a transcription factor in the ETS family, is characterized by several distinct domains and motifs that are crucial for its function. The ETS domain, responsible for DNA binding, is a prominent feature, consisting of three alpha-helices and four anti-parallel beta-strands, forming an abbaabb pattern (Hussain2011Molecular). This domain includes a nuclear localization signal (NLS) and a nuclear export signal (NES), essential for its cellular localization (Besnard2011Elk-1).

ELK1 also contains a B box for interaction with serum response factor (SRF), a transactivation domain rich in phosphorylation sites, and a DEF domain that specifically docks with ERK, a type of MAP kinase (Ducker2019De-ubiquitination; Besnard2011Elk-1). Post-translational modifications of ELK1 include phosphorylation, SUMOylation, and ubiquitylation, which regulate its activity and stability (Ducker2019De-ubiquitination; Besnard2011Elk-1).

Additionally, ELK1 exists in several splice variant isoforms, including âElk-1 and sElk-1, which differ in their interaction domains and specific functions within neuronal contexts (Besnard2011Elk-1). These structural variations and modifications are critical for ELK1's role in transcriptional regulation and its response to cellular signals.

## Function
ELK1, an ETS-domain transcription factor, plays a pivotal role in transducing extracellular signals into transcriptional responses, primarily through the MAP kinase signaling pathway. It functions by binding to promoter regions of target genes, often near the transcription start site, which suggests its role in the initial stages of transcription regulation (Boros2009Elucidation). ELK1 is involved in the regulation of over 1000 target genes, including core components of the gene regulation machinery such as basal transcription complex components, spliceosome subunits, and ribosomal proteins (Boros2009Elucidation).

The transcription factor is activated in response to various extracellular signals including growth factors and stress, mediated by different MAP kinases like ERK, JNK, and p38. Once activated, ELK1 binds more effectively to DNA and recruits coactivators like CBP and p300, enhancing its transcriptional activity. This process is crucial for the transcription of immediate early genes (IEGs) and other genes involved in core gene expression control (Besnard2011Elk-1).

Additionally, ELK1 plays a crucial role in cellular functions related to the actin cytoskeleton and cell migration. It controls the activity of specific target genes within these functional categories, significantly influencing cell migration and cytoskeletal characteristics (Odrowaz2012ELK1). This underscores ELK1's essential role in maintaining normal cell morphology and motility, highlighting its broad impact on cellular function and organismal outcomes.

## Clinical Significance
ELK1, an ETS transcription factor, has been implicated in various diseases due to mutations, altered expression levels, or disruptions in its normal interactions. In prostate cancer, ELK1 functions as a tethering protein for the androgen receptor, facilitating the activation of growth genes. Its expression correlates with higher clinical and pathology T-stage, Gleason score, prognostic grade, and positive lymph node status, serving as an independent negative prognosticator of disease-free survival (DFS) (Pardy2019The). 

In Huntington's disease, ELK1 plays a protective role against mutant Huntingtin toxicity. Reducing ELK1 expression exacerbates toxicity, while its overexpression, particularly a mutated, constitutively active form, reduces aggregate formation and enhances neuronal survival. This suggests therapeutic potential in modulating gene expression and reducing cellular toxicity associated with the disease (Yildirim2019Early).

Furthermore, ELK1 is involved in the regulation of the EVI1 gene in acute myeloid leukemia (AML) and influences the expression of the oncogenic long non-coding RNA MIR100HG in osteosarcoma, impacting disease progression and patient survival (Su2019ELK1-induced; Maicas2012Functional). These findings underscore the clinical significance of ELK1 in various cancers and neurodegenerative disorders, highlighting its role in disease mechanisms and potential as a target for therapeutic intervention.

## Interactions
ELK1, an ETS-domain transcription factor, engages in various physical interactions with other proteins and nucleic acids, which are crucial for its function in transcriptional regulation. ELK1 forms dimers and potentially larger oligomeric complexes such as tetramers, which are important for its activity in cells (Drewett2000Dimer). It interacts with ERK2 MAP kinase through its D domain, a critical interaction for the phosphorylation and activation of ELK1, which enhances its DNA-binding capacity and recruitment into ternary complexes with serum response factor (SRF) and the c-fos SRE (Yang1998The). Additionally, ELK1 can bind to DNA at sites containing the GGAA/T motif, and this binding is modulated by its interactions with SRF (Odrowaz2012ELK1).

ELK1 also interacts with the MZF-1 protein through specific domains that influence their mutual DNA-binding activities and transcriptional regulation functions (Lee2016MZF-1/Elk-1). Furthermore, ELK1's ability to form ternary complexes is enhanced by phosphorylation at specific serine residues, which is mediated by interactions with MAP kinases (Gille1995ERK). These interactions underscore the multifaceted role of ELK1 in regulating gene expression in response to various cellular signals.


## References


[1. (Boros2009Elucidation) Joanna Boros, Ian J. Donaldson, Amanda OâDonnell, Zaneta A. Odrowaz, Leo Zeef, Mathieu Lupien, Clifford A. Meyer, X. Shirley Liu, Myles Brown, and Andrew D. Sharrocks. Elucidation of the elk1 target gene network reveals a role in the coordinate regulation of core components of the gene regulation machinery. Genome Research, 19(11):1963â1973, August 2009. URL: http://dx.doi.org/10.1101/gr.093047.109, doi:10.1101/gr.093047.109. (152 citations) 10.1101/gr.093047.109](https://doi.org/10.1101/gr.093047.109)

[2. (Besnard2011Elk-1) Antoine Besnard, Beatriz Galan-Rodriguez, Peter Vanhoutte, and Jocelyne Caboche. Elk-1 a transcription factor with multiple facets in the brain. Frontiers in Neuroscience, 2011. URL: http://dx.doi.org/10.3389/fnins.2011.00035, doi:10.3389/fnins.2011.00035. (224 citations) 10.3389/fnins.2011.00035](https://doi.org/10.3389/fnins.2011.00035)

[3. (Pardy2019The) Luke Pardy, Rayna Rosati, Claire Soave, Yanfang Huang, Seongho Kim, and Manohar Ratnam. The ternary complex factor protein elk1 is an independent prognosticator of disease recurrence in prostate cancer. The Prostate, 80(2):198â208, December 2019. URL: http://dx.doi.org/10.1002/pros.23932, doi:10.1002/pros.23932. (13 citations) 10.1002/pros.23932](https://doi.org/10.1002/pros.23932)

[4. (Odrowaz2012ELK1) Zaneta Odrowaz and Andrew D. Sharrocks. Elk1 uses different dna binding modes to regulate functionally distinct classes of target genes. PLoS Genetics, 8(5):e1002694, May 2012. URL: http://dx.doi.org/10.1371/journal.pgen.1002694, doi:10.1371/journal.pgen.1002694. (82 citations) 10.1371/journal.pgen.1002694](https://doi.org/10.1371/journal.pgen.1002694)

[5. (Yildirim2019Early) Ferah Yildirim, Christopher W. Ng, Vincent Kappes, Tobias Ehrenberger, Siobhan K. Rigby, Victoria Stivanello, Theresa A. Gipson, Anthony R. Soltis, Peter Vanhoutte, Jocelyne Caboche, David E. Housman, and Ernest Fraenkel. Early epigenomic and transcriptional changes reveal elk-1 transcription factor as a therapeutic target in huntingtonâs disease. Proceedings of the National Academy of Sciences, 116(49):24840â24851, November 2019. URL: http://dx.doi.org/10.1073/pnas.1908113116, doi:10.1073/pnas.1908113116. (35 citations) 10.1073/pnas.1908113116](https://doi.org/10.1073/pnas.1908113116)

[6. (Su2019ELK1-induced) Xiaochuan Su, Junyan Teng, Guoguo Jin, Jitian Li, Zhenjiang Zhao, Xiangyang Cao, Yanxing Guo, Malong Guo, Xiaoling Li, Jun Wu, Chuanzhen Wang, Zhiping Guo, and Qing Guo. Elk1-induced upregulation of long non-coding rna mir100hg predicts poor prognosis and promotes the progression of osteosarcoma by epigenetically silencing lats1 and lats2. Biomedicine &amp; Pharmacotherapy, 109:788â797, January 2019. URL: http://dx.doi.org/10.1016/j.biopha.2018.10.029, doi:10.1016/j.biopha.2018.10.029. (51 citations) 10.1016/j.biopha.2018.10.029](https://doi.org/10.1016/j.biopha.2018.10.029)

[7. (Ducker2019De-ubiquitination) Charles Ducker, Leo Kam Yuen Chow, Janice Saxton, JÃ¼rgen Handwerger, Alexander McGregor, Thomas Strahl, Robert Layfield, and Peter E Shaw. De-ubiquitination of elk-1 by usp17 potentiates mitogenic gene expression and cell proliferation. Nucleic Acids Research, 47(9):4495â4508, March 2019. URL: http://dx.doi.org/10.1093/nar/gkz166, doi:10.1093/nar/gkz166. (30 citations) 10.1093/nar/gkz166](https://doi.org/10.1093/nar/gkz166)

[8. (Maicas2012Functional) M Maicas, I VÃ¡zquez, C Vicente, M A GarcÃ­a-SÃ¡nchez, N Marcotegui, L Urquiza, M J Calasanz, and M D Odero. Functional characterization of the promoter region of the human evi1 gene in acute myeloid leukemia: runx1 and elk1 directly regulate its transcription. Oncogene, 32(16):2069â2078, June 2012. URL: http://dx.doi.org/10.1038/onc.2012.222, doi:10.1038/onc.2012.222. (37 citations) 10.1038/onc.2012.222](https://doi.org/10.1038/onc.2012.222)

[9. (Drewett2000Dimer) Victoria Drewett, Silke Muller, Jane Goodall, and Peter E. Shaw. Dimer formation by ternary complex factor elk-1. Journal of Biological Chemistry, 275(3):1757â1762, January 2000. URL: http://dx.doi.org/10.1074/jbc.275.3.1757, doi:10.1074/jbc.275.3.1757. (31 citations) 10.1074/jbc.275.3.1757](https://doi.org/10.1074/jbc.275.3.1757)

[10. (Lee2016MZF-1/Elk-1) Chia-Jen Lee, Li-Sung Hsu, Chia-Herng Yue, Ho Lin, Yung-Wei Chiu, Yu-Yu Lin, Chih-Yang Huang, Mien-Chie Hung, and Jer-Yuh Liu. Mzf-1/elk-1 interaction domain as therapeutic target for protein kinase cÎ±-based triple-negative breast cancer cells. Oncotarget, 7(37):59845â59859, August 2016. URL: http://dx.doi.org/10.18632/oncotarget.11337, doi:10.18632/oncotarget.11337. (23 citations) 10.18632/oncotarget.11337](https://doi.org/10.18632/oncotarget.11337)

[11. (Hussain2011Molecular) Abrar Hussain, Peter E Shaw, and Jonathan D Hirst. Molecular dynamics simulations and in silico peptide ligand screening of the elk-1 ets domain. Journal of Cheminformatics, November 2011. URL: http://dx.doi.org/10.1186/1758-2946-3-49, doi:10.1186/1758-2946-3-49. (6 citations) 10.1186/1758-2946-3-49](https://doi.org/10.1186/1758-2946-3-49)

[12. (Yang1998The) Shen-Hsi Yang, Paula R. Yates, Alan J. Whitmarsh, Roger J. Davis, and Andrew D. Sharrocks. The elk-1 ets-domain transcription factor contains a mitogen-activated protein kinase targeting motif. Molecular and Cellular Biology, 18(2):710â720, February 1998. URL: http://dx.doi.org/10.1128/MCB.18.2.710, doi:10.1128/mcb.18.2.710. (344 citations) 10.1128/MCB.18.2.710](https://doi.org/10.1128/MCB.18.2.710)

[13. (Gille1995ERK) H. Gille, M. Kortenjann, O. Thomae, C. Moomaw, C. Slaughter, M.H. Cobb, and P.E. Shaw. Erk phosphorylation potentiates elk-1-mediated ternary complex formation and transactivation. The EMBO Journal, 14(5):951â962, March 1995. URL: http://dx.doi.org/10.1002/j.1460-2075.1995.tb07076.x, doi:10.1002/j.1460-2075.1995.tb07076.x. (524 citations) 10.1002/j.1460-2075.1995.tb07076.x](https://doi.org/10.1002/j.1460-2075.1995.tb07076.x)