# LEF1

## Overview
LEF1, or lymphoid enhancer-binding factor 1, is a gene that encodes a transcription factor crucial for various cellular processes, including cell proliferation, differentiation, and apoptosis. The protein encoded by this gene, also referred to as LEF1, is a member of the high mobility group (HMG) box family of DNA-binding proteins and plays a pivotal role in the Wnt signaling pathway. LEF1 functions by interacting with Î²-catenin to regulate the transcription of target genes involved in cell fate decisions and developmental processes. The protein is characterized by its HMG domain, which facilitates specific DNA binding and bending, essential for its function as a transcriptional regulator. LEF1's involvement in critical signaling pathways and its interactions with other proteins underscore its importance in developmental biology and its implications in various diseases, including cancer and hematological disorders (LÃ©vy2020LEF1; Zhou2008LEF-1).

## Structure
LEF1 (lymphoid enhancer-binding factor 1) is a protein characterized by a High Mobility Group (HMG) domain that is crucial for its DNA-binding ability. This HMG box, spanning amino acids 296 to between 365 and 380, is essential for sequence-specific DNA interactions, particularly with the nucleotide sequence 5'-CCTTTGAA within the T-cell receptor alpha enhancer. The HMG domain does not display obvious structural features like amphipathic helices but includes regions that could potentially form alpha-helices (Giese1991DNA-binding).

Additionally, LEF1 interacts with Î²-catenin through its N-terminal domain in the Wnt/Î²-catenin signaling pathway. This domain binds to Î²-catenin's armadillo (arm) domain, which consists of twelve armadillo repeats forming an elongated superhelix of Î± helices. The interaction involves a characteristic amino acid sequence motif and includes non-polar contacts and salt bridges (Sun2011Biochemical). Phosphorylation by Casein Kinase II at Ser 40 in the N-terminal domain does not alter its binding affinity with Î²-catenin, suggesting that post-translational modifications at this site do not significantly impact the structural integrity or the interaction dynamics of the LEF1-Î²-catenin complex (Sun2011Biochemical).

The structural details of LEF1 also highlight the disordered nature of certain regions, such as residues 26-41, which include glutamates that could potentially form salt bridges in other TCF structures but are disordered in the non-phosphorylated LEF1 complex (Sun2011Biochemical). These features underscore the complex and dynamic nature of LEF1's structure, which is critical for its function in cellular signaling and gene regulation.

## Function
LEF1 (lymphoid enhancer-binding factor 1) is a transcription factor that plays a significant role in various cellular processes including cell proliferation, differentiation, and apoptosis. It is part of the high mobility group (HMG) box family of DNA-binding proteins and functions in a context-dependent manner, particularly in the regulation of the T cell receptor (TCR) alpha enhancer. LEF1 is considered an 'architectural' transcription factor, helping in the assembly of a multiprotein enhancer complex, which includes other lymphoid-specific proteins such as Ets-1, PEBP2-alpha, and ATF/CREB (Zhou2008LEF-1).

LEF1 mediates Wnt signals by recruiting Î²-catenin to Wnt response elements (WREs) of target genes, a process crucial for cell fate decisions, proliferation, survival activities, and morphological changes. This interaction is facilitated by the N-terminal Î²-catenin-binding domain of LEF1, which undergoes conformational changes to bind Î²-catenin effectively. The high-mobility group (HMG) DNA-binding domain of LEF1 recognizes specific DNA sequences, enhancing its binding affinity and enabling it to bend DNA, which is critical for the regulation of gene expression (Arce2006Diversity).

In the nucleus, LEF1's activity is regulated by its nuclear localization signal, which facilitates its import into the nucleus by importin alpha subunits. This localization is essential for its function in transcriptional regulation and its ability to interact with other transcription factors and proteins, influencing gene expression and cellular outcomes (Arce2006Diversity).

## Clinical Significance
LEF1 (lymphoid enhancer-binding factor 1) mutations and altered expression levels are implicated in various hematological disorders and cancers. In acute myeloid leukemia (AML), high LEF1 expression is associated with better relapse-free survival and overall survival, particularly in patients with intermediate-risk cytogenetics who are wild-type for FLT3-ITD. LEF1 expression, combined with mutation status of FLT3-ITD or NPM1, is used in a scoring system to predict patient outcomes in AML (Fu2013Clinical). Conversely, in chronic lymphocytic leukemia (CLL), high LEF1 expression correlates with unmutated IGHV status, indicating a worse prognosis and shorter treatment-free survival (Wu2016High). 

LEF1 also plays a role in congenital neutropenia, where mutations or altered expression can lead to a maturation arrest of granulopoiesis at the promyelocyte stage due to defective G-CSFR signaling (Skokowa2012Interactions). Additionally, LEF1 haploinsufficiency, as evidenced by a 4q25 de novo deletion, has been linked to severe oligodontia, hypotrichosis, and hypohidrosis, symptoms characteristic of hypohidrotic ectodermal dysplasia (LÃ©vy2020LEF1).

These findings underscore the clinical significance of LEF1 in both promoting and inhibiting disease progression, depending on the context of its expression and genetic alterations.

## Interactions
LEF1 interacts with several proteins to regulate transcriptional activities, crucially involving Î²-catenin, which lacks intrinsic DNA-binding capability and relies on LEF1 to facilitate gene expression regulation. The interaction between LEF1 and Î²-catenin is mediated through the N-terminal beta-catenin-binding domain of LEF1, which engages Î²-catenin through induced conformational changes, forming an alpha helix and salt bridges in the interaction groove of Î²-catenin's Armadillo repeat domain (Arce2006Diversity). This interaction is essential for the transcriptional regulation activities of LEF1 in the Wnt signaling pathway.

LEF1 also interacts with HDAC1, a histone deacetylase, which plays a role in transcriptional repression. This interaction does not compete with Î²-catenin for binding sites on LEF1, allowing LEF1 to form complexes with both HDAC1 and Î²-catenin simultaneously, which facilitates LEF1's role in switching between transcriptional repression and activation (Billin2000Î²-CateninâHistone).

Additionally, LEF1 can bind to gamma-catenin/plakoglobin, although this interaction is weaker and occurs through a region C-terminal to the beta-catenin-binding domain (Arce2006Diversity). LEF1 also interacts with TLE1, which can repress LEF-1/Î²-catenin-mediated transcriptional activation by physically interacting with LEF-1, possibly preventing LEF-1 from effectively engaging with Î²-catenin or the transcription machinery (Levanon1998Transcriptional). These interactions highlight the complex role of LEF1 in transcriptional regulation, involving multiple protein interactions that contribute to its function in developmental processes and possibly in cancer.


## References


[1. (Levanon1998Transcriptional) Ditsa Levanon, Robert E. Goldstein, Yael Bernstein, Hua Tang, Dalia Goldenberg, Stefano Stifani, Zeâev Paroush, and Yoram Groner. Transcriptional repression by aml1 and lef-1 is mediated by the tle/groucho corepressors. Proceedings of the National Academy of Sciences, 95(20):11590â11595, September 1998. URL: http://dx.doi.org/10.1073/pnas.95.20.11590, doi:10.1073/pnas.95.20.11590. (593 citations) 10.1073/pnas.95.20.11590](https://doi.org/10.1073/pnas.95.20.11590)

[2. (Skokowa2012Interactions) Julia Skokowa, Maxim Klimiankou, Olga Klimenkova, Dan Lan, Kshama Gupta, Kais Hussein, Esteban Carrizosa, Inna Kusnetsova, Zhixiong Li, Claudio Sustmann, Arnold Ganser, Cornelia Zeidler, Hans-Heinrich Kreipe, Janis Burkhardt, Rudolf Grosschedl, and Karl Welte. Interactions among hcls1, hax1 and lef-1 proteins are essential for g-csfâtriggered granulopoiesis. Nature Medicine, 18(10):1550â1559, September 2012. URL: http://dx.doi.org/10.1038/nm.2958, doi:10.1038/nm.2958. (89 citations) 10.1038/nm.2958](https://doi.org/10.1038/nm.2958)

[3. (Fu2013Clinical) Yuan Fu, Huayuan Zhu, Wei Wu, Jiadai Xu, Tingmei Chen, Bei Xu, Sixuan Qian, Jianyong Li, and Peng Liu. Clinical significance of lymphoid enhancer-binding factor 1 expression in acute myeloid leukemia. Leukemia &amp; Lymphoma, 55(2):371â377, July 2013. URL: http://dx.doi.org/10.3109/10428194.2013.805759, doi:10.3109/10428194.2013.805759. (47 citations) 10.3109/10428194.2013.805759](https://doi.org/10.3109/10428194.2013.805759)

[4. (Arce2006Diversity) L Arce, N N Yokoyama, and M L Waterman. Diversity of lef/tcf action in development and disease. Oncogene, 25(57):7492â7504, December 2006. URL: http://dx.doi.org/10.1038/sj.onc.1210056, doi:10.1038/sj.onc.1210056. (569 citations) 10.1038/sj.onc.1210056](https://doi.org/10.1038/sj.onc.1210056)

[5. (LÃ©vy2020LEF1) Jonathan LÃ©vy, Yline Capri, Myriam Rachid, CÃ©line Dupont, Joris R. Vermeesch, Koen Devriendt, Alain Verloes, AnneâClaude Tabet, and Isabelle BailleulâForestier. Lef1 haploinsufficiency causes ectodermal dysplasia. Clinical Genetics, 97(4):595â600, February 2020. URL: http://dx.doi.org/10.1111/cge.13714, doi:10.1111/cge.13714. (18 citations) 10.1111/cge.13714](https://doi.org/10.1111/cge.13714)

[6. (Wu2016High) Wei Wu, Huayuan Zhu, Yuan Fu, Wenyi Shen, Kourong Miao, Min Hong, Wei Xu, Lei Fan, Ken H. Young, Peng Liu, and Jianyong Li. High lef1 expression predicts adverse prognosis in chronic lymphocytic leukemia and may be targeted by ethacrynic acid. Oncotarget, 7(16):21631â21643, February 2016. URL: http://dx.doi.org/10.18632/oncotarget.7795, doi:10.18632/oncotarget.7795. (46 citations) 10.18632/oncotarget.7795](https://doi.org/10.18632/oncotarget.7795)

[7. (Billin2000Î²-CateninâHistone) Andrew N. Billin, Hilary Thirlwell, and Donald E. Ayer. Î-cateninâhistone deacetylase interactions regulate the transition of lef1 from a transcriptional repressor to an activator. Molecular and Cellular Biology, 20(18):6882â6890, September 2000. URL: http://dx.doi.org/10.1128/MCB.20.18.6882-6890.2000, doi:10.1128/mcb.20.18.6882-6890.2000. (278 citations) 10.1128/MCB.20.18.6882-6890.2000](https://doi.org/10.1128/MCB.20.18.6882-6890.2000)

[8. (Giese1991DNA-binding) K Giese, A Amsterdam, and R Grosschedl. Dna-binding properties of the hmg domain of the lymphoid-specific transcriptional regulator lef-1. Genes &amp; Development, 5(12b):2567â2578, December 1991. URL: http://dx.doi.org/10.1101/gad.5.12b.2567, doi:10.1101/gad.5.12b.2567. (196 citations) 10.1101/gad.5.12b.2567](https://doi.org/10.1101/gad.5.12b.2567)

[9. (Zhou2008LEF-1) Fangfang Zhou, Long Zhang, Kai Gong, Guangyuan Lu, Baiyang Sheng, Aijun Wang, Nanming Zhao, Xiufang Zhang, and Yandao Gong. Lef-1 activates the transcription of e2f1. Biochemical and Biophysical Research Communications, 365(1):149â153, January 2008. URL: http://dx.doi.org/10.1016/j.bbrc.2007.10.138, doi:10.1016/j.bbrc.2007.10.138. (45 citations) 10.1016/j.bbrc.2007.10.138](https://doi.org/10.1016/j.bbrc.2007.10.138)

[10. (Sun2011Biochemical) Jinglucy Sun and William I. Weis. Biochemical and structural characterization of Î²-catenin interactions with nonphosphorylated and ck2-phosphorylated lef-1. Journal of Molecular Biology, 405(2):519â530, January 2011. URL: http://dx.doi.org/10.1016/j.jmb.2010.11.010, doi:10.1016/j.jmb.2010.11.010. (59 citations) 10.1016/j.jmb.2010.11.010](https://doi.org/10.1016/j.jmb.2010.11.010)